### PR TITLE
feat(dca): Style positions card

### PR DESCRIPTION
**Pull request by th0rOdinson**
### Closed position Card
Screenshot 2024-03-11 at 14 47 16

### Open Position Card
Screenshot 2024-03-11 at 14 47 27

### Open Position Card Options
Screenshot 2024-03-11 at 14 53 20

### CODE REVIEW

File: packages/ui-library/src/theme/variants/tooltip-variants.ts [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
+import { baseColors, colors } from '../colors';
+import { SPACING } from '../constants';
 
 export const buildTooltipVariant = (mode: 'light' | 'dark'): Components => ({
   MuiTooltip: {
     styleOverrides: {
       tooltip: {
-        backgroundColor: `${colors[mode].background.emphasis} !important`,
-        borderColor: colors[mode].border.border1,
+        backgroundColor: `${colors[mode].background.emphasis}`,
```
Comment (FiboApe) [inline packages/ui-library/src/theme/variants/tooltip-variants.ts:9]: no need for quotes

File: packages/ui-library/src/components/position-progress-bar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+export const PositionProgressBar = withStyles(
+  ({ variant = 'determinate', ...props }: LinearProgressProps) => <LinearProgress variant={variant} {...props} />,
+  ({ palette: { mode } }, { value }) => ({
+    bar: {
+      background:
+        value === 100
+          ? colors[mode].semantic.success.darker
+          : mode === 'light'
+          ? `linear-gradient(270deg, ${colors.light.violet.violet500} 0%, ${colors.light.violet.violet400} ${value}%)`
```
Comment (FiboApe) [inline packages/ui-library/src/components/position-progress-bar/index.tsx:14]: we could probably define a variable for this on the theme like "linear progress"

File: packages/ui-library/src/components/options-menu/index.tsx [hunk 3]
```diff
@@ -116,13 +118,13 @@ const OptionsMenuItems = ({ options, handleClose, anchorEl }: OptionsMenuItemsPr
...
                   <Typography
                     variant="bodyExtraSmall"
-                    color={itemColor ? `${itemColor}.main` : colors[mode].typography.typo3}
+                    color={itemColor ? `${itemColor}.dark` : colors[mode].typography.typo3}
                   >
                     {secondaryLabel}
                   </Typography>
                 )}
               </ContainerBox>
-              {control && <Box sx={{ marginLeft: 'auto' }}>{control}</Box>}
```
Comment (FiboApe) [inline packages/ui-library/src/components/options-menu/index.tsx:125]: this doesnt break other stuff?

File: apps/root/src/pages/dca/positions/components/positions-list/current-positions/index.tsx [hunk 4]
```diff
@@ -197,12 +187,7 @@ const CurrentPositions = ({ isLoading }: CurrentPositionsProps) => {
     }
   };
 
-  const positionsInProgress = currentPositions
-    .filter(({ toWithdraw, remainingSwaps }) => toWithdraw > 0n || remainingSwaps > 0n)
-    .sort(comparePositions);
-  const positionsFinished = currentPositions
-    .filter(({ toWithdraw, remainingSwaps }) => toWithdraw <= 0n && remainingSwaps <= 0n)
-    .sort(comparePositions);
+  const sortedPositions = orderBy(currentPositions, ['remainingSwaps', 'toWithdraw'], ['desc', 'desc']);
```
Comment (FiboApe) [inline apps/root/src/pages/dca/positions/components/positions-list/current-positions/index.tsx:190]: Have you checked that this orders correctly? Had some issue with orderBy and bigints where it converted them to strings before, so it was doing string comparison

File: apps/root/src/pages/dca/positions/components/positions-list/position-card/index.tsx [hunk 5]
```diff
@@ -0,0 +1,535 @@
...
+            {isTestnet && (
+              <ContainerBox alignItems="flex-start">
+                <Chip
+                  label={<FormattedMessage description="testnet" defaultMessage="Testnet" />}
+                  size="small"
+                  color="warning"
+                />
+              </ContainerBox>
+            )}
+            {((position.from.symbol === 'CRV' && foundYieldFrom) || (position.to.symbol === 'CRV' && foundYieldTo)) && (
```
Comment (FiboApe) [inline apps/root/src/pages/dca/positions/components/positions-list/position-card/index.tsx:423]: Can you move all of this warning messages into its own component? so its easier to modify in the future

File: packages/ui-library/src/components/position-progress-bar/index.tsx (hunk 2 above)
Comment (th0rOdinson) [reply packages/ui-library/src/components/position-progress-bar/index.tsx:14]: At first I did! But I noticed that the last % value is dynamic. We could do a function but I don't think it's going to be included as a theme color

File: apps/root/src/pages/dca/positions/components/positions-list/position-card/index.tsx (hunk 5 above)
Comment (th0rOdinson) [reply apps/root/src/pages/dca/positions/components/positions-list/position-card/index.tsx:423]: Absolutely!

File: packages/ui-library/src/components/options-menu/index.tsx (hunk 3 above)
Comment (th0rOdinson) [reply packages/ui-library/src/components/options-menu/index.tsx:125]: I've tested all implementations of this component and works fine, I found this style redundant. The label container is already fullWidth, so this control should always be pushed to the right

**Review by FiboApe — APPROVED** [review]

**Pull request merged**